### PR TITLE
[DX-1625] Fix 404s from old pages per Google search console

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!

[DX-1625](https://tyktech.atlassian.net/browse/DX-1625)

---

In 4.2 we had a directory with & in the path. Hugo version at that time dropped it. https://tyk.io/docs/4.2/tyk-cloud/account--billing/plans/
In 4.3 we moved it to a different location without an & - https://tyk.io/docs/4.3/tyk-cloud/account-billing/plans/
In 5.2 or 5.3 we made a change and added `canonical` to all our pages hoping Google will stop referring to docs of specific versions, and just use our latest. This means that the URL in the old release 4.2 https://tyk.io/docs/4.2/tyk-cloud/account--billing/plans/ redirect to https://tyk.io/docs/tyk-cloud/account--billing/plans/, but since we changed the path, the crawling generates 404. We also did not have the automation in 4.3, but now if we remove a URL we get an error and we know we have to add an alias. 

### Preview Link

<be>

### Tested 
1. https://deploy-preview-5268--tyk-docs.netlify.app/docs/nightly/tyk-cloud/account--billing/plans/
2. https://deploy-preview-5268--tyk-docs.netlify.app/docs/nightly/tyk-cloud/account--billing/retirement/
3. https://deploy-preview-5268--tyk-docs.netlify.app/docs/nightly/tyk-rest-api/hot-reload/
4. https://deploy-preview-5268--tyk-docs.netlify.app/docs/nightly/concepts/what-is-a-session-object/

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

[DX-1625]: https://tyktech.atlassian.net/browse/DX-1625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/tyk-cloud/account-billing/plans.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ menu:
 weight: 1
 aliases:
   - /tyk-cloud/account-and-billing/our-plans/
+  - tyk-cloud/account--billing/plans/
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/account-billing/plans.md]:
```suggestion
  - /tyk-cloud/account--billing/plans/
```

**Review by dcs3spp — APPROVED** [review]
PR LGTM, will merge and release

**Pull request merged**